packages/examples/tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "extends": "../../tsconfig.json",
     "compilerOptions": {
         "outDir": "dist",
-        "module": "commonjs",
+        "module": "Node16",
```

Comment:
@relu91 you kept `"commonjs"` because of scrambled output, right? Do you mind changing it?

Reply:
Yes, this should be kept, any reason for changing it @erossignon ?